requirements-dev.txt

```
@@ -0,0 +1,4 @@
+-r requirements.txt
+flake8==3.7.7
+mock==2.0.0
+pytest==4.6.3
```

I know in other repos we avoid using pytest 5 because it breaks our tests for some reason, but I think if we're adding tests for the first time we should probably try using 5 straight out the gates?

```suggestion
pytest==5.4.3
```

---

Ah this broke the Travis tests. Of course pytest 5 doesn't support Python 2. My bad 🤦

---

Hah, I forgot as well 😹